src/script/run_tox.sh

```diff
@@ -115,11 +115,20 @@ function main() {
             shift
         fi
         tox_path=$(get_tox_path $test_name)
+        if [ -z "$test_name" ]; then
```

Comment:
i'd suggest refactor the code to guess tox_path and test_name:
```sh
if [ $# -gt 0 ]; then
  test_name=$1
  shift
  tox_path=$(find $source_dir -path "*/$test_name/tox.ini" | xargs -I{} dirname {})
elif [ -e tox.ini ]; then
  tox_path=$(pwd)
else
  echo "--tox-path not specified" >& 2
  exit 2
fi
```

Reply:
> i'd suggest refactor the code to guess tox_path and test_name:
> 
> ```shell
> if [ $# -gt 0 ]; then
>   test_name=$1
>   shift
>   tox_path=$(find $source_dir -path "*/$test_name/tox.ini" | xargs -I{} dirname {})
> elif [ -e tox.ini ]; then
>   tox_path=$(pwd)
> else
>   echo "--tox-path not specified" >& 2
>   exit 2
> fi
> ```

If I'm not mistaken, this still doesn't properly set `test_name`, if it is empty and not given, right?